src/components/main-view/main-view.jsx

```diff
@@ -85,7 +87,6 @@ console.log(error);
        /* If there is no user, the LoginView is rendered. If there is a user 
         logged in, the user details are *passed as a prop to the LoginView*/
 
-        if (movies.length === 0) return <div className="main-view" />;
```

Comment:
This line is no longer necessary since the `.map` loop won't render any movies when the array is empty.